### DISCUSSION TITLE
Hello all Can you help me return the money? I sent Ethereum to buy Manta but I have not received the coins for a month yet

### DISCUSSION THREAD

**Discussion by MOIUJIBKHALIL** [blockscout/blockscout#9432] [General]
ETH
-0.25
ETH
Transaction Details

Blockchain
Ethereum

Status
Success

Timestamp
last month
(Jan 17, 2024 at 06:19 pm)

Block Number
[19028044](https://etherscan.io/block/19028044)

Block Hash
0x856a...a0d2bc

Value
0.250315636825044 ETH

Transaction Index

Gas Price
41751201684

Gas Used
21000

Nonce
0

**Comment by MOIUJIBKHALIL** [comment]
Etherscan logo
Jan 17, 2024 at 06:19 pm
From (You)
0x144a...c863d1
(0x144...63d1)
To
0x95ce...c544e5
(0x95c...44e5)
Transaction Hash
0xcfa1...ce4f98
Transaction Fee
0.001 ETH
Sent
ETH
-0.25
ETH
Transaction Details

Blockchain
Ethereum

Status
Success

Timestamp
last month
(Jan 17, 2024 at 06:19 pm)

Block Number
[19028044](https://etherscan.io/block/19028044)

Block Hash
0x856a...a0d2bc

Value
0.250315636825044 ETH

Transaction Index

Gas Price
41751201684

Gas Used
21000

Nonce
0

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
